File: Backend/fuel-app-server/src/main/java/com/dimitriskikidis/fuelappserver/fuel/FuelSearchRequest.java
package com.dimitriskikidis.fuelappserver.fuel;

import java.util.List;

public record FuelSearchRequest(
        List<Integer> brandIds,
        Integer fuelTypeId,
        Double minLatitude,
        Double maxLatitude,
        Double minLongitude,
        Double maxLongitude
) {
}

File: Android/FuelApp/app/src/main/java/com/dimitriskikidis/fuelapp/data/remote/requests/FuelSearchRequest.kt
package com.dimitriskikidis.fuelapp.data.remote.requests

data class FuelSearchRequest(
    val brandIds: List<Int>,
    val fuelTypeId: Int,
    val minLatitude: Double,
    val maxLatitude: Double,
    val minLongitude: Double,
    val maxLongitude: Double
)
